### PR TITLE
[autoscaler][kubernetes][minor] latest images everywhere

### PR DESCRIPTION
**Pull request by DmitriGekhtman**
## Why are these changes needed?

Final bit of uniformity with the rest of the Ray repo.

Todo: proper version management, pinned images in releases, host a Helm repo.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: deploy/charts/ray/values.yaml [hunk 1]
```diff
@@ -71,9 +71,5 @@ namespacedOperator: false
 # operatorNamepsace - If using a cluster-scoped operator (namespacedOperator: false), set the namespace
 # in which to launch the operator.
 operatorNamespace: default
-# operatorImage - For best stability, it's currently recommended to use the nightly ray image in the operator Deployment.
-# For an official ray release version, use `rayproject/ray:1.x.y`.
```
Comment (pcmoritz) [inline deploy/charts/ray/values.yaml:75]: Do we still have these instructions somewhere in the docs? They seem useful :)

**Review by pcmoritz — APPROVED** [review]

File: deploy/charts/ray/values.yaml (hunk 1 above)
Comment (DmitriGekhtman) [reply deploy/charts/ray/values.yaml:75]: Images are documented in the installation docs, but doesn't hurt to be redundant.

**Comment by DmitriGekhtman** [comment]
please merge, thx

**Comment by richardliaw** [comment]
(rubber stamp merge)

**Pull request merged**